omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -234,6 +234,20 @@
 default['private_chef']['opscode-erchef']['couchdb_max_conn'] = '100'
 default['private_chef']['opscode-erchef']['ibrowse_max_sessions'] = 256
 default['private_chef']['opscode-erchef']['ibrowse_max_pipeline_size'] = 1
+# general search settings used to set up chef_index
+default['private_chef']['opscode-erchef']['search_provider'] = "solr" # solr, TODO cloudsearch
+default['private_chef']['opscode-erchef']['search_queue_mode'] = "rabbitmq" # rabbitmq, batch, or inline
```

Comment:
Can we change the unit, or accept a unit indicator to make it more clear? "5M" or "5MB" vs "5000000"; or `search_batch_max_mb`?